lib/wanda/messaging/adapters/amqp.ex

```diff
@@ -5,6 +5,16 @@ defmodule Wanda.Messaging.Adapters.AMQP do
 
   @behaviour Wanda.Messaging.Adapters.Behaviour
 
+  alias Wanda.Messaging.Adapters.AMQP.Publisher
+
+  @impl true
+  def publish(routing_key, message), do: Publisher.publish_message(message, routing_key)
```

Comment:
Isn't it a bit strange that in the 1st signature we have the `routing_key` 1st and the message 2nd, and in the second function just in the opposite order?

Reply:
This is to mimic what phoenix channel does, which has the topic as first parameter and the message as the second. We used this in trento web so I would vote to keep the (topic, message) order in this way.